salt/modules/zypper.py

```diff
@@ -628,16 +628,16 @@ def install(name=None,
         log.info('Targeting repo {0!r}'.format(fromrepo))
     else:
         fromrepoopt = ''
+    cmd_install = ['zypper', '--no-refresh', '--non-interactive', 'install',
```

Comment:
@rominf Please add here `--no-refresh` the same way as `--download-only`. If we want it by default with no refresh, then we should add parameter `refresh=False`. Also please consider refresh the DB by calling `refresh_db()`.